src/api.ts

```diff
@@ -84,6 +87,39 @@ export const getRecommendations = async (page: number, perPage: number, sort: st
   return response;
 };
 
+/* Remediations */
+
+export const postRemediations = async (path: string, data) => {
+  const response = await axios.post(REMEDIATIONS_API_ROOT.concat(path), data).catch(function (error) {
+    return error;
+  });
+
+  return getResponseOrError(response);
+};
+
+export const remediationsResolutions = (ruleId: string) => {
```

Comment:
Was kinda confused from the name, basically it is just about getting the info about given rule, right?

Reply:
Yes, the endpoint is named resolutions, so I used that as the name for the function.